io.catenax.batch/2.0.0/Batch.ttl

```diff
@@ -0,0 +1,180 @@
+:KeyValueList a samm:Entity;
+    samm:preferredName "Key Value List"@en;
+    samm:description "A list of key value pairs for local identifiers, which are composed of a key and a corresponding value."@en;
+    samm:properties (:key :value).
+:ManufacturingEntity a samm:Entity;
+    samm:preferredName "Manufacturing Entity"@en;
+    samm:description "Encapsulates the manufacturing relevant attributes"@en;
+    samm:properties (:date [
+  samm:property :country;
+  samm:optional "true"^^xsd:boolean
+]).
+:PartTypeInformationEntity a samm:Entity;
+    samm:preferredName "Part Type Information Entity"@en;
+    samm:description "Encapsulation for data related to the part type"@en;
+    samm:properties (:manufacturerPartId :nameAtManufacturer :classification).
+:key a samm:Property;
+    samm:preferredName "Identifier Key"@en;
+    samm:description "The key of a local identifier. "@en;
+    samm:characteristic :KeyTrait;
+    samm:exampleValue "BatchID".
+:value a samm:Property;
+    samm:preferredName "Identifier Value"@en;
+    samm:description "The value of an identifier."@en;
+    samm:characteristic :ValueCharacteristic;
+    samm:exampleValue "BID12345678".
+:date a samm:Property;
+    samm:preferredName "Production Date"@en;
+    samm:description "Timestamp of the manufacturing date as the final step in production process (e.g. final quality check, ready-for-shipment event)"@en;
+    samm:characteristic samm-c:Timestamp;
+    samm:exampleValue "2022-02-04T14:48:54"^^xsd:dateTime.
+:country a samm:Property;
+    samm:preferredName "Country code"@en;
+    samm:description "Country code where the part was manufactured"@en;
+    samm:characteristic :ProductionCountryCodeTrait;
+    samm:exampleValue "HUR".
+:manufacturerPartId a samm:Property;
+    samm:preferredName "Manufacturer Part ID"@en;
+    samm:description "Part ID as assigned by the manufacturer of the part. The Part ID identifies the part (as designed) in the manufacturer`s dataspace. The Part ID does not reference a specific instance of a part and thus should not be confused with the serial number or batch number."@en;
+    samm:characteristic :PartIdCharacteristic;
+    samm:exampleValue "123-0.740-3434-A".
+:nameAtManufacturer a samm:Property;
+    samm:preferredName "Name at Manufacturer"@en;
+    samm:description "Name of the part as assigned by the manufacturer"@en;
+    samm:characteristic :PartNameCharacteristic;
+    samm:exampleValue "Mirror left".
+:classification a samm:Property;
+    samm:preferredName "Classifcation"@en;
+    samm:description "The classification of the part type according to STEP standard definition"@en;
+    samm:characteristic :ClassificationCharacteristic;
+    samm:exampleValue "raw material".
+:KeyTrait a samm-c:Trait;
+    samm:preferredName "Key Trait "@en;
+    samm:description "Trait that ensures the usage of predefined keys. "@en;
```

Comment:
remove whitespace at the end of the string

Reply:
done